README.md

```diff
@@ -150,9 +150,11 @@ This code uses an additional feature of finalizers: unregistration. It works lik
 
 Unregistration is useful here to avoid holding a reference to the file past when it was already closed, to avoid closing it multiple times. Imagine--once the file's been closed, the same file descriptor may have been reused for something unrelated, and closing it a second time could mess up unrelated code!
 
-### Avoiding more memory leaks
+### Avoid memory leaks for cross-worker proxies
```

Comment:
Nitpicking but you won't actually avoid all memory leaks since finalizers can't detect cycle between the workers ;)

Reply:
PRs welcome to make this more specific.

Reply:
I went for a new section on reference cycles instead: #106